src/Components/Blazor/WebAssembly.Authentication/src/Interop/tsconfig.json

```diff
@@ -0,0 +1,65 @@
+{
+  "compilerOptions": {
```

Comment:
Could we simplify this just to the things we are using? No need to document the rest of the config file format :)

Reply:
Done